packages/apps/desktop-viewer-test/src/frontend/components/modelSelector/SelectIModel.tsx

```diff
@@ -199,9 +200,10 @@ export const SelectIModel = ({
 
   return (
     <div className="itv-scrolling-container select-imodel">
```

Comment:
are any of these custom classes we created being used anymore / still relevant with the introduction of the layout pkg?